tests/provision_basic_el.sh

```diff
@@ -31,4 +31,9 @@ puppet resource file /etc/puppetlabs/code/environments/production/modules/sensu
 
 # setup module dependencies
 puppet module install puppetlabs/stdlib --version 4.25.1
+puppet module install puppet/trusted_ca --version 2.0.0
+
+puppet resource host puppet ensure=present ip=192.168.52.10
+puppet resource host sensu-backend.example.com ensure=present ip=192.168.52.10
+puppet agent --test --tags no_such_tag
```

Comment:
could you use `-t` instead of `--test`. Most people learn with `-t` as the name 'test' is vary misleading.

Reply:
Changed.